### DISCUSSION TITLE
Improve info about number of episodes left in progress screen

### DISCUSSION THREAD

**Discussion by Matth7878** [trakt/showly#686] [Ideas / Feature Requests]
Option for next episode is really a good thing and it allows you to choose between last watched or oldest to calculate it.

But when you use last watched then number of episodes left should also reflect your choice.
For instance for a show with 24 episodes with : 
- 1 episode left after last watched
- 7 in total not watched
- 16 watched
then progress screen shouldn't just tell you 8 left. User probably won't watch episodes he skipped.

So imho it should tell you there is only 1 episode left. Or better it should be something like : 16/24 (1 left, 7 skipped)

**Comment by michaldrabik** [comment]
One general problem with skipping episodes is that Trakt does not recognize such a functionality so it would be impossible to sync that.

**Reply by Matth7878** [reply]
Could it be just a calculation? I thought left episodes was calculated looking at episodes marked unplayed.
If it is not wouldn't it be possible to do it this way even if it is something not synced?

The app already do something similar for next episode as you can choose to use oldest unwatched or last episode watched to calculate it. And I think It's also something no synced?

**Reply by michaldrabik** [reply]
In theory that may work, will need to check it out.
